File: wwwroot/js/models/inventoryItemType.js
var InventoryItemType = function (data) {
    var self = this;
    if (data) {
        ko.mapping.fromJS(data, {}, self);
    }
    else {
        self.name = ko.observable();
        self.unit = ko.observable();
        self.unitPrice = ko.observable();
        self.quantity = ko.observable();
    }
}

File: Domain/InventoryItemType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace MessingSystem.Domain
{
    public class InventoryItemType
    {
        public int ItemTypeId { get; set; }
        public string Name { get; set; }
        public string Unit { get; set; }
        public decimal UnitPrice { get; set; }
        public decimal Quantity { get; set; }

    }

}
